src/js/components/ClipboardTrigger.js

```diff
@@ -44,39 +43,59 @@ class ClipboardTrigger extends React.Component {
     }
   }
 
+  getTriggerContent() {
+    const {children, className} = this.props;
+
+    if (children != null) {
+      return (
+        <span className={className} onMouseEnter={this.handleCopyIconMouseEnter}
+          ref="copyButton">
+          {children}
+        </span>
+      );
+    }
+
+    return (
+      <Icon
```

Comment:
Apparently the `Clipboard` constructor will not accept an `<svg />` element as a first argument. This means that we should wrap the `<Icon>` with a `<span>` or something else, and put the `ref=` there.